pygmt/tests/test_clib_to_numpy.py

```diff
@@ -18,6 +18,18 @@
 
     _HAS_PYARROW = True
 except ImportError:
+
+    class pa:  # noqa: N801
```

Comment:
`pa.timestamp()` with `tz` doesn't have string aliases, so we can't use strings like `timestamp[s, UTC]`. So we have to define a dummy function here.

Reply:
We can, but I feel it will make the pytest parametrize more complicated.

Reply:
Ok, let's keep this dummy class then.